plugins/package-managers/python/src/main/kotlin/utils/PythonInspector.kt

```diff
@@ -39,6 +39,8 @@ private val json = Json {
     namingStrategy = JsonNamingStrategy.SnakeCase
 }
 
+internal const val DEFAULT_PYTHON_VERSION = "3.11"
```

Comment:
I don't think this is the better place, because this class does not enforce any default. Only `Pip` does. So essentially this is moving the constant farther away from the only place that uses it.